main/recom.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pandas as pd
+from lockfile import LockFile
+from time import sleep
+
+
+def start():
+    if not os.path.exists('recom_data.csv'):
+        with open('recom_data.csv', 'w') as fp:
+            fp.write("Files")
+
+
+lock = LockFile('recom_data.csv')
+
+
+def get(user_id):

```

Comment:
Rename this function to something more informational, like get_recs() or get_recommendations() etc.